doc/lua_api.txt

```diff
@@ -1535,19 +1553,22 @@ Tools define:
         * Digging times
         * Damage groups
 
-#### Full punch interval
+Full punch interval
+-------------------
 When used as a weapon, the tool will do full damage if this time is spent
 between punches. If e.g. half the time is spent, the tool will do half
 damage.
 
-#### Maximum drop level
+Maximum drop level
+------------------
 Suggests the maximum level of node, when dug with the tool, that will drop
 it's useful item. (e.g. iron ore to drop a lump of iron).
 
 This is not automated; it is the responsibility of the node definition
 to implement this.
 
-#### Uses
+Uses
```

Comment:
This is actually `###` a level below `Tools definition`as it describes a part of the tool definition.